wrappers/python/tests/layer/test_batch_norm.py

```diff
@@ -0,0 +1,97 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/tests/layer/test_layer_norm.py
+# Test for nntile.layer.LayerNorm
+#
+# @version 1.0.0
+
+import nntile
+from nntile.layer import BatchNorm2d
+
+import nntile.utils
+import nntile.utils.constructors
+import numpy as np
+import torch
+import torch.nn as nn
+from dataclasses import dataclass
+
+config = nntile.starpu.Config(1, 0, 0)
+
+nntile.starpu.init()
+dtypes = [np.float32, np.float64]
+
+@dataclass
+class BatchNormTestParams:
+    shape: list
```

Comment:
Usually, shape is inmmutable so it should be a tuple.

Reply:
done